pkg/result/ignore.go

```diff
@@ -106,8 +106,11 @@ func (f *IgnoreFindings) Match(id string, results *FindingsResults) *IgnoreFindi
 			continue
 		}
 
-		log.Debug("Ignored", log.String("id", id), log.String("target", results.TargetPath),
-			log.String("startLine-endLine", fmt.Sprintf("%d-%d", results.StartLine, results.EndLine)))
+		log.Debug("Ignored", log.String("id", id), log.String("target", results.TargetPath))
+
+		if finding.StartLine > 0 || finding.EndLine > 0 {
```

Comment:
Shouldn't this be zero inclusive? What if start or end are actually 0?

Reply:
The only place I found references to a possible usage of 0 in a test for either start or end line were here: https://github.com/aquasecurity/trivy/blob/main/pkg/report/table/misconfig_test.go#L75-L76. In this scenario the start and end line checks would be skipped.

There are also references to `>0` here: https://github.com/aquasecurity/trivy/blob/main/pkg/report/table/misconfig.go#L161

Reply:
You're right.